Resources/AzureDevOps/JobTemplates/deploy-environment.yml

```diff
@@ -0,0 +1,230 @@
+parameters:
+  AzureSubscription: 'SFA-CDH-Dev/Test (962cae10-2950-412a-93e3-d8ae92b17896)'  
+  SetSubscriptionScript: 'https://raw.githubusercontent.com/SkillsFundingAgency/dfc-devops/master/PSScripts/Set-EsfaResourceGroupTags.ps1'  
+  PackageFile:  ''
+  ArmTemplateFolder: ''
+  ApimImportScript: ''
+  SpecificationPath: ''
+  Location: 'West Europe'
+  ArmOutputVariableName: 'ARMOutputs'
+  EnvironmentTag: 'DEV/Test'
+  ParentBusiness: ''
```

Comment:
ParentBusiness and ServiceOffering is unlikely to change so could have a default set

Reply:
Not quite, confusingly we have National Careers Service (PP) for every environment apart from production and National Careers Service for production. Similarly with ServiceOffering there is (PP) alternatives for all non-production sites

Reply:
I've altered this to use the values from the variable groups